### ISSUE TITLE
7daa273 / #87 broke convGDX2mif for old.gdx

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [pik-piam/remind#95]
[Testing](https://github.com/pik-piam/remind/blob/182f0a504fa6a52307ab07eaf82e2c4b13e5a152/tests/testthat/test-convGDX2mif.R) the remind package with [`old.gdx`](https://github.com/pik-piam/remind/blob/182f0a504fa6a52307ab07eaf82e2c4b13e5a152/inst/extdata/old.gdx) results in 
```
── Failure (test-convGDX2mif.R:48:3): Test if REMIND reporting is produced as it
`runParallelTests(my_gdxs)` threw an error.
Message: subscript out of bounds ("FE|Transport|Pass|Liquids|Biomass (EJ/yr)")
```
Offending line: https://github.com/pik-piam/remind/blob/fa0b7622ca2bef8e160a4bd8ade66066b25e52c1/R/reportCrossVariables.R#L146

**Comment by Loisel** [comment]
Should be fixed in #96

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
> Should be fixed in #96

So you didn't test it? 🤣

**Comment by Loisel** [comment]
This time it just *has* to work :P
